### PR TITLE
[1LP][RFR] Adding service dialog multi drop test

### PR DESCRIPTION
**Pull request by niyazRedhat**
### PRT Run
{{pytest: cfme/tests/services/test_dialog_element_in_catalog.py::test_save_dynamic_multi_drop_down_dialog }}

### CODE REVIEW

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 1]
```diff
@@ -1136,3 +1138,41 @@ def test_dialog_default_value_selection(appliance, custom_categories, import_dat
         view.fields("network").read() == network and
         view.fields("number_disk").read() == additional_disks
     )
+
+
+@pytest.mark.meta(automates=[1559030])
+@pytest.mark.tier(2)
+@pytest.mark.customer_scenario
```
Comment (ganeshhubale) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: remove tier marker

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 2]
```diff
@@ -1136,3 +1138,41 @@ def test_dialog_default_value_selection(appliance, custom_categories, import_dat
...
+        testtype: functional
+        initialEstimate: 1/4h
+        startsin: 5.10
+    """
+    sd, ele_label = import_dialog
+    navigate_to(sd, "Edit")
+
+    # update dialog element
+    view = appliance.browser.create_view(EditElementView)
+    view.element.edit_element(ele_label)
```
Comment (ganeshhubale) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: Isn't this navigation(navigate_to(sd, "Edit")) helps to reach at this page - EditElementView?

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 3]
```diff
@@ -1136,3 +1138,41 @@ def test_dialog_default_value_selection(appliance, custom_categories, import_dat
...
+
+    # enable multi select button
+    view.options.multi_select.fill("Yes")
+    view.ele_save_button.click()
+    view.save_button.click()
+
+    # no error should be displayed
+    view = sd.create_view(DetailsDialogView)
+    view.wait_displayed()
+    view.flash.assert_success_message(f'{sd.label} was saved')
```
Comment (ganeshhubale) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: I think: you can add ```wait``` in create_view(DetailsDialogView). No need of   ```view.wait_displayed()```.

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 4]
```diff
@@ -1136,3 +1138,41 @@ def test_dialog_default_value_selection(appliance, custom_categories, import_dat
...
+    view.flash.assert_no_error()
```
Comment (ganeshhubale) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: Why do we need this assertion as we already checking assert message?

File: cfme/tests/services/test_dialog_element_in_catalog.py (hunk 2 above)
Comment (niyazRedhat) [reply cfme/tests/services/test_dialog_element_in_catalog.py]: No, we have to navigate by EditElementView

File: cfme/tests/services/test_dialog_element_in_catalog.py (hunk 3 above)
Comment (niyazRedhat) [reply cfme/tests/services/test_dialog_element_in_catalog.py]: there is some issue with `wait` . `view.wait_displayed()` would be nice here.

Comment (ganeshhubale) [reply cfme/tests/services/test_dialog_element_in_catalog.py]: can you please point that issue here? Is GH issue created for it?

Comment (niyazRedhat) [reply cfme/tests/services/test_dialog_element_in_catalog.py]: not sure the issue but i heard that and there is one PR of Parthvi https://github.com/ManageIQ/integration_tests/pull/9978/files#diff-1af1dca8d2ad3e8627bac286076744ebR491

Comment (ganeshhubale) [reply cfme/tests/services/test_dialog_element_in_catalog.py]: @valaparthvi any issue you know related to **wait**?

Comment (valaparthvi) [reply cfme/tests/services/test_dialog_element_in_catalog.py]: I don't think there is any problem with create_view's `wait`. That problem was with FlashMessages widget. I think the widget stopped supporting `wait` argument.

Comment (ganeshhubale) [reply cfme/tests/services/test_dialog_element_in_catalog.py]: Thanks @valaparthvi @niyazRedhat

**Pull request merged**